recipe/yum_requirements.txt

```diff
@@ -1 +1,4 @@
 mesa-libGL
+libX11
```

Comment:
I don't think this file is actually used by prefect.
However the issue with missing libX11 got solved on August 5.

Reply:
I did not know that; will revert this change (adding this file did fix the issue though).